waterwheel/waterwheel.py

```diff
@@ -0,0 +1,109 @@
+import re
+import os
+import pickle
+import json
+import pandas as pd
+from tqdm import tqdm
+from pathlib import Path
+from typing import Dict, List
+
+import spacy
+from spacy.matcher import PhraseMatcher
+from spacy.tokens import Doc, Span, DocBin
+
+from .wheel_util import read_files
+
+
+class WaterWheel():
+    """Class for finding hydrologic entities in text"""
+
+    def __init__(self, nlp):
+        """Initialize the class.
+        
+        Parameters
+        ----------
+        nlp :
+            spacy nlp object
+        """
+
+        self.matcher = None
+        #self.nlp = spacy.load('en_core_web_sm')
+        self.nlp = nlp
+        self.vocab = {}
+        self.wikidata = {}
+        self.stop_words = set()
```

Comment:
there is a convention in Python to name private vars of a class with leading underscore so `self._nlp` and then 
having 
```
@property
def nlp(self):
     return self._nlp
```
for variables we want to expose on the class. This is just to be more explicit. Can we please add `TODO: make class variables "private" with underscores`?

Reply:
Added TODO.